modyn/evaluator/internal/dataset/evaluation_dataset.py

```diff
@@ -34,6 +34,8 @@ def __init__(
         storage_address: str,
```

Comment:
Now that the evaluation dataset is becoming more and more different from the onlinedataset, I wonder whether we should have a separate EvaluationDataset integrationtest, similar to test_online_dataset.py. Does not need to be part of this PR, but maybe we should create an issue to write a test here

Reply:
I didn't get it? I think this PR just augments two storage APIs, i.e. `GetDataPerWorkerRequest` and `GetDatasetSizeRequest`, which are on the storage server side, so the current integration tests are in `integrationtest_storage.py`.

Reply:
I extended issue #275. Just a note, (integration) tests are not necessarily driven by complexity. In a perfect world, unit tests mock away everything except the function-under-test. The idea of an integration test is to verify whether a component works end-to-end. They are two different perspectives on testing. Anyways, nothing that affects this PR.